### DISCUSSION TITLE
Intercept back press , even if only one screen in the stack

### DISCUSSION THREAD

**Discussion by aicameron10** [adrielcafe/voyager#300] [Q&A]
Hi , I wanted to know if it's possible to intercept the back press , even if only one screen in the stack , and back would exit the application, currently, you can see the current screen, when more then one screen in the stack and you back press you see the last screen key , is it possible to intercept if only one screen ?

**Comment by Syer10** [comment]
Why wouldn't this work?
```
BackHandler(navigator.size == 1) {
    // Whatever you want to do
}
```

**Reply by aicameron10** [reply]
BackHandler is not yet available in multiple compose unfortunately, is an android only function I beleive
